### ISSUE TITLE
Local domain names stopped working after upgrade

### ISSUE THREAD

**Issue by dshatovskiy** [symfony-cli/symfony-cli#95]
Hi guys,

I have a problem using local domain names after upgrade on the new version (5.3.4) from symfony/cli deprecated v4.x
Before following install instructions and install symfony-cli from rpm (my OS is Fedora) I have dropped .symfony dir in home and binary file from /usr/local/bin/

I have started proxy, installed certificates, attached domain name and started server.
The issue - browser does not load the page by domain name, it stuck for a minute and breaks with timeout. The same page via  https://127.0.0.1:port works just fine.
I have also tried to do the same without certificates uninstalling  - the same problem.

I was also trying to remove everything and install 4.x version back but nothing changed, unfortunately I didn't notice exact version before deleting it. 

Also noticed, when I start a new server or when I hit that unreachable link, proxy becomes dead, address http://localhost:7080/ is not reachable for some period of time, then after few minutes it starts working again

This is what I have found in proxy.log
`"time":"2022-02-17T17:17:01+02:00","message":"[048] WARN: Cannot write TLS response body from mitm'd client: write tcp 127.0.0.1:7080->127.0.0.1:40208: write: connection reset by peer"}
{"time":"2022-02-17T17:43:30+02:00","message":"[046] WARN: Error responding to client: write tcp 127.0.0.1:7080->127.0.0.1:45578: write: broken pipe"}`

Please help to troubleshoot this.
Thank you for your help

**Comment by phcorp** [comment]
Yes custom domains seem not to be added to the generated certificate. Currently I'm using `mkcert` and `openssl` to replace the certificate generated by the symfony CLI:
```shell
# generate a certificate for 'my-domain.tld'
CAROOT="${HOME}/.symfony5/certs" mkcert -pkcs12 -p12-file temp.p12 "my-domain.tld" localhost 127.0.0.1 ::1
# convert the p12 certificate to pem to remove mkcert default 'changeit' password
openssl pkcs12 -in temp.p12 -nodes -out temp.pem -passin pass:changeit -legacy && rm temp.p12
# save the p12 certificate in place of symfony's default one
openssl pkcs12 -export -in temp.pem -out "${HOME}/.symfony5/certs/default.p12" -passout pass: && rm temp.pem
```

**Comment by tucksaun** [comment]
> custom domains seem not to be added to the generated certificate.

This is because certificates for custom domaines are generated on the fly by the proxy.

**Comment by tucksaun** [comment]
@dshatovskiy it clearly seems your issue lies in the proxy connection hanging.
Can you try to run the proxy manually in foreground with a more verbose logging level (eg: `symfony local:proxy:start -f -v=4`)?

**Comment by tucksaun** [comment]
@phcorp yours issues can be explained by the fact that you edited the certificates. I invite you to stop all the processes (proxy and webserver), removes the certificate (`default.p12`), run `symfony local:server:ca:install` and try again.

**Comment by tucksaun** [comment]
@dshatovskiy I believe your issue lies somewhere else so I'm still interested in your logs if possible :)

**Comment by phcorp** [comment]
@tucksaun thanks! I'm not sure if it's because I upgraded to the latest version (`v5.8.14`, I was previously on `v5.8.13`) but the problem does not appear anymore

**Comment by tucksaun** [comment]
@dshatovskiy are you still experiencing this issue?
